### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -200,7 +200,7 @@ endif()
 include(ProcessorCount)
```
Comment (xansec) [inline CMakeLists.txt]: Do we have to comment this out? Or can we wrap it in an if-gate for fuzzing?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: CMakeLists.txt (hunk 1 above)
Comment (rnshah9) [reply CMakeLists.txt]: Forgot to uncomment! It just sped up the build.

**Comment by xansec** [comment]
LGTM!

**Pull request merged**